Scripts/script-CPCreateBackup.yml

```diff
@@ -39,18 +45,19 @@ script: |
       except Exception as ex:
           res.append({"Type": entryTypes["error"], "ContentsFormat": formats["text"],
                       "Contents": "Error occurred while parsing output from command. Exception info:\n" + str(ex) + "\n\nInvalid output:\n" + str(resSSH)})
-      demisto.setContext('chkpbackupstarted', devicesBackupStarted )
-      demisto.setContext('devicesbackuperrorstarting', devicesBackupError )
+      demisto.setContext('CheckpointBackup', devicesBackupStarted )
       res.append({"Type": entryTypes["note"], "ContentsFormat": formats["table"], "Contents": tbl})
   demisto.results(res)
+
 type: python
 tags:
 - checkpoint
 comment: Connect to a checkpoint firewall appliance using SSH and trigger a task to
   create a configuration backup of the device. The user account being used to access
   the device must be set to use the SSH shell and not the built in Checkpoint CLI.
   Consult the Checkpoint documentation for instructions on how to do this.
-system: true
+enabled: true
+deprecated: true
```

Comment:
I changed it so it would add the correct things to context, just in case anyone uses it. It's the same code as CheckpointFWCreateBackup